lib/rubocop/config_loader.rb

```diff
@@ -81,25 +84,29 @@ def merge(base_hash, derived_hash)
         result
       end
 
-      def base_configs(path, inherit_from)
+      def base_configs(path, inherit_from, file)
         configs = Array(inherit_from).compact.map do |f|
-          if f =~ /\A#{URI::Parser.new.make_regexp(%w[http https])}\z/
-            f = RemoteConfig.new(f, File.dirname(path)).file
-          else
-            f = File.expand_path(f, File.dirname(path))
-
-            if auto_gen_config?
-              next if f.include?(AUTO_GENERATED_FILE)
-            end
-
-            print 'Inheriting ' if debug?
-          end
-          load_file(f)
+          f = load_inherited(path, f, file)
+          load_file(f) unless f.nil?
```

Comment:
Could `#load_file` handle the `nil` case instead? Otherwise any consumer of the method needs to know that it requires manual `nil` checks. 🙂